mapping_form.html

```diff
@@ -0,0 +1,150 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <title>MANYC Airtable-Gateway Mapping Form</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```

Comment:
Also not for this PR, but I'd love if we could pull down these resources via npm and import them locally [as in the mapping repo](https://github.com/MutualAidNYC/mapping/blob/539b53be58e1131fb938df945824a57b193298af/server/index.js#L69-L91), rather than use CDNs, which can be a privacy concern.